app/forms/sufia/forms/work_form.rb

```diff
@@ -7,5 +7,17 @@ def rendered_terms
                :visibility_after_embargo, :visibility_during_lease,
                :lease_expiration_date, :visibility_after_lease, :visibility, :thumbnail_id, :representative_id, :ordered_member_ids]
     end
+
+    # override Curation Concerns Work Form, Sufia allows multiple values for rights
+    def self.multiple?(term)
+      case term.to_s
+      when 'rights'
+        true
+      when 'ordered_member_ids'
```

Comment:
@jcoyne @mjgiarlo the reason this method override is required is because without it the `rights` field is being rejected by the controller. This is the method in CC that Nabeela is overriding: https://github.com/projecthydra-labs/curation_concerns/blob/master/app/forms/curation_concerns/forms/work_form.rb#L44

I suspect the fact that the field is defined as multi-value but the CC `multiple?` implementation marks it as single-value causes a mismatch that prevents the controller from allowing this field during the HTTP POST request.

@njaffer I think writing the override as follow might make it clearer that the intention is to handle a specific issue for the `rights` field:

```
def self.multiple?(term)
  return true if term == :rights
  super
end
```